src/runtime/components/overlays/Notifications.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <Teleport to="body">
+  <Teleport v-if="notifications.length" to="body">
```

Comment:
Is the `v-if` not better on the root div? 🤔

Reply:
yes actually it's better but the `notifications` value is define in the component `Notifications` it self. or should i add to root ?